index.html

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
```

Comment:
index.html and index1.html is deleted :/ Revert them.

Reply:
@pumanitro DONE